### PR TITLE
Add Chinese doc of AdadeltaOptimizer

### PR DESCRIPTION
**Pull request by sneaxiy**
Related to [Paddle#19875](https://github.com/PaddlePaddle/Paddle/pull/19875).

image

23d9247465fa8b7c3fc966b637a21c66

### CODE REVIEW

**Comment by zq19** [comment]
请将PR填写到API整改修复进度表，方便统计进度。

File: doc/fluid/api_cn/optimizer_cn/AdadeltaOptimizer_cn.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    learning\_rate &= \sqrt{ ( E(dx_{t-1}^2) + \epsilon ) / ( E(g_t^2) + \epsilon ) }\\
+    E(dx_t^2) &= \rho * E(dx_{t-1}^2) + (1-\rho) * (-g*learning\_rate)^2
+
+
+参数：
+    - **learning_rate** (float|Variable) - 全局学习率。
+    - **epsilon** (float) - 维持数值稳定性的浮点型值，默认值为1.0e-6。
+    - **rho** (float) - 算法中的衰减率，默认值为0.95。
+    - **regularization** (WeightDecayRegularizer，可选) - 正则化方法，例如fluid.regularizer.L2DecayRegularizer等。默认值为None，表示无正则化。
+    - **name** (str，可选) – 该参数供开发人员打印调试信息时使用，具体用法请参见 :ref:`api_guide_Name` ，默认值为None。
```
Comment (lanxianghit) [inline doc/fluid/api_cn/optimizer_cn/AdadeltaOptimizer_cn.rst:26]: ref:`api_guide_Name` 
请确认有效性，预览图中未显示链接样式

Comment (sneaxiy) [reply doc/fluid/api_cn/optimizer_cn/AdadeltaOptimizer_cn.rst:26]: Done.

Comment (sneaxiy) [reply doc/fluid/api_cn/optimizer_cn/AdadeltaOptimizer_cn.rst:26]: 刚才没pull最新develop分支，所以链接在预览图里没生效，已更新预览图，请查阅。

**Review by lanxianghit — APPROVED** [review]
LGTM

**Pull request merged**